app/code/Ludmila/LSNewClasses/Model/ShowTypes.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Ludmila\LSNewClasses\Model;
+
+class showTypes
+{
+
+    public $typeString;
```

Comment:
Why these properties are public? It will work with public ones, but please specify the reason of making them public (by commenting the code or in github comment). Please specify where they are accessed as public ones?